src/test/commons.bash

```diff
@@ -6,8 +6,18 @@ BIN_DIR="$THIS/../../src/main"
 MOCK_DIR="$THIS/../../mock"
 export PATH=$MOCK_DIR:$BIN_DIR:$PATH
 
+check(){
+    run "$@"
+    echo "> Exit code: \$status=$status"
+    for line in ${lines[@]}; do
+        echo "> stdout+stderr: '$line'"
+     done
```

Comment:
After this loop in output we have each word of executed command output on separate line, not line by line.

Reply:
Fixed